test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -6961,6 +6961,77 @@ public async Task RestoreNetCore_SingleTFM_SameIdSameVersionMultiDeclaration_Mul
             }
         }
 
+        [Fact]
```

Comment:
There are already tests covering this scenario so you might need to amend those. 

I'd imagine they'd fail with your change.

Reply:
Actually they won't, cause you didn't remove the old behavior.